sql/core/src/main/scala/org/apache/spark/sql/SQLContext.scala

```diff
@@ -991,6 +991,33 @@ class SQLContext(@transient val sparkContext: SparkContext)
   }
 
   /**
+   * :: Experimental ::
+   * Construct a [[DataFrame]] including [0, n) of LongType, with column name as `id`.
+   *
+   * @group dataframe
```

Comment:
when you update this, please make sure you add "@since 1.4.0" to this. also do it for the other range function. thanks!